src/client/inputhandler.h

```diff
@@ -129,6 +131,15 @@ class MyEventReceiver : public IEventReceiver
 		return b;
 	}
 
+	// Checks whether a key was released and resets the state
+	bool WasKeyReleased(const KeyPress &keyCode)
```

Comment:
This function name is deceptive as it does more than the function name implies (i.e. it checks whether a key was release -- this is the implied behaviour -- *and* it resets the state which is the unexpected side-effect not reflected in the function name.